src/cluster_legacy.c

```diff
@@ -5419,6 +5433,28 @@ static void clusterNodeCronFreeLinkOnBufferLimitReached(clusterNode *node) {
     freeClusterLinkOnBufferLimitReached(node->inbound_link);
 }
 
+/**
+ * Compute the maximum number of connection attempts the cluster-cron
+ * loop should schedule in a single cron.
+ *
+ * We want to guarantee that every node is contacted 10 times within node timeout.
+ */
+static long long maxConnectionAttemptsPerCron(void) {
+    long long reconnect_interval = server.cluster_node_timeout / 2;
+    if (dictSize(server.cluster->nodes) <= 0 || reconnect_interval <= 0)
+        return 0;
+    /*
+     * We run the cron loop every 100 ms.  To reach 100 % of the nodes
+     * within the timeout, we need: ceil(nodes * 100 / reconnect_interval)
+     */
+    const long long min_nodes_for_coverage = ceil(dictSize(server.cluster->nodes) * CLUSTER_CRON_PERIOD_MS / reconnect_interval);
```

Comment:
All of these are integers, so ceil will not do anything since it will always be an integer
```suggestion
    const long long min_nodes_for_coverage = dictSize(server.cluster->nodes) * CLUSTER_CRON_PERIOD_MS / reconnect_interval;
```